staging/src/k8s.io/kubectl/pkg/cmd/version/skew_warning.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import (
+	"fmt"
+	"io"
+	apimachineryversion "k8s.io/apimachinery/pkg/version"
+	"regexp"
+	"strconv"
+)
+
+const supportedMinorVersionSkew = 1
```

Comment:
Do we have a consensus into this? To be honest, as Kubernetes supports at least 3 minor versions, (like from 1.20 to 1.18) I'm afraid of maybe this issuing a warning because of a small difference, but this is my opinion :)

Reply:
+-1 version does support 3 versions -- 1.19 supports 1.18, 1.19, 1.20

Reply:
True! Will re-read what I've wrote on the other review to see if still makes sense

Reply:
Let me know if you determine something different.  I added a comment explaining which versions would fall within the supported skew range.